src/SingleCellAnnotator.imjoy.html

```diff
@@ -404,20 +409,31 @@
       // Default options are marked with *
       const sendData = JSON.parse(JSON.stringify(sendDataTemplate));
       sendData.discarded = this.current_image_data.discarded;
-      sendData.mask = this.current_image_data.mask_geojson.features;
-      sendData.bbox = this.current_image_data.mask_geojson.bbox;
-      sendData.location = {};
-      if (this.current_image_data.label_display_mode == 'single'){
-        sendData.location[this.current_image_data.location] = this.current_location_selected;
-        } else if (this.current_image_data.label_display_mode == 'multiple'){
-          sendData.location = this.current_image_data.location;
-          for (loc in sendData.location){
-            if(sendData.location[loc]==null){
-              sendData.location[loc] = false}
+      if (this.current_image_data.discarded){
+        // delete sendData.mask;
+        sendData.mask = null;
+      }else{
+        sendData.bbox = this.current_image_data.mask_geojson.bbox;
+        sendData.location = {};
+        if (this.current_image_data.label_display_mode == 'single'){
+          sendData.location[this.current_image_data.location] = this.current_location_selected;
+          } else if (this.current_image_data.label_display_mode == 'multiple'){
+            sendData.location = this.current_image_data.location;
+            for (loc in sendData.location){
+              if(sendData.location[loc]==null){
+                sendData.location[loc] = false}
+            }
+            // add things here
+          } else{api.alert("label_display_mode parameter out of range");
+          return };
+        if (this.current_image_data.mask_adjusted==true){
+          sendData.mask = this.current_image_data.mask_geojson.features;
+          this.current_image_data.mask_adjusted==false;
```

Comment:
It should be: `this.current_image_data.mask_adjusted=false;`

Reply:
corrected in the new PR